self-contained/bootstrap_install.sh

```diff
@@ -67,7 +67,7 @@ installNetworkPlugin() {
     rm -rf $CNI_DOWNLOADS_DIR &
 }
 
-
+# Cameron do you want these functions to match?
```

Comment:
@cameronmeissner Is it fine if the functions in bootstrap_install.sh are diverging from what is in cse_install.sh?

Reply:
ideally they should remain in alignment, yeah

will make @Devinwong's and @lilypan26's lives a lot easier